tabular/tests/unittests/test_tabular.py

```diff
@@ -974,3 +974,19 @@ def test_tabular_bagstack_use_bag_holdout():
     ###################################################################
     run_tabular_benchmarks(fast_benchmark=fast_benchmark, subsample_size=subsample_size, perf_threshold=perf_threshold,
                            seed_val=seed_val, fit_args=fit_args, run_distill=True, crash_in_oof=True)
+
+
+def test_tabular_raise_on_nonfinite_labels():
```

Comment:
What about `["a", "b", np.nan]`? Or is this not possible?

Reply:
This was a great question. I wasn't sure if/how `pandas` and `numpy` would handle that. I found a better way to handle this case as well and pushed an update with a new Unit Test to show it correct fails on this case as well.